pandera/backends/pyspark/decorators.py

```diff
@@ -81,6 +81,22 @@ def validate_scope(scope: ValidationScope):
     def _wrapper(func):
         @functools.wraps(func)
         def wrapper(self, *args, **kwargs):
+            def _get_check_obj():
+                """
+                Get dataframe object passed as arg to the decorated func.
+
+                Returns:
+                    The DataFrame object.
+                """
+                if kwargs:
+                    for value in kwargs.values():
+                        if isinstance(value, pyspark.sql.DataFrame):
+                            return value
```

Comment:
Code coverage does not reach this because we are not passing `check_obj` as a kwarg. We use an arg instead.

Reply:
- Can you confirm if there's any scenario where a DataFrame might be passed as a kwarg?
- For our tests related to this decorator, are we ensuring that we pass the pyspark.sql.DataFrame both as a positional and as a keyword argument?
- If we anticipate that the DataFrame will always be passed as a positional argument and never as a keyword argument, would it make sense to refactor the decorator to remove the kwargs check for simplicity?

Reply:
I wrongly assumed that this decorator was used in `pandas` and other namespaces too and I wanted to keep the pattern, but checking now, there is no such thing in other integrations.
I'm changing it to remove the `kwargs` capability, as it's not used anywhere. Thank you for the input.

Reply:
decorator was added to `pyspark` only. I hope eventually `pandas` will follow the suite...